index.js

```diff
@@ -267,9 +283,6 @@ client.on("message", async message => {
       message.channel.send(reminders[rand]);
     } else if (command.match(/\blofi\b/) != null) {
       message.channel.send("https://open.spotify.com/playlist/1DcvziAZBZk1Ji1c65ePtk?si=Qtvu64zsQQurDtQa60tPBg");
-    } else if (command.match(/\bping\b/) != null) {
```

Comment:
Why not send the embed in this snippet here??
`message.channel.send({embed: {
        color: 0x2ed32e,
        fields: [{
            name: "Your Ping",
            value: "My Ping: " + Math.round(client.ws.ping) + ' ms'
      }
     ],
}`

Reply:
yup thats a better idea lmao